sea-query-derive/tests/pass/3_level_flatten.rs

```diff
@@ -1,7 +1,7 @@
-use sea_query::Iden;
+use sea_query::{Iden, IdenStatic};
 use strum::{EnumIter, IntoEnumIterator};
 
-#[derive(Iden, EnumIter)]
+#[derive(Iden, IdenStatic, EnumIter)]
```

Comment:
Out of curiosity, can we derive `IdenStatic` without deriving `Iden`?

Reply:
No, we cannot...

Reply:
I think this is possible if we also generate the following bit:

```rust
impl Iden for Character {
    fn unquoted(&self, s: &mut dyn std::fmt::Write) {
        write!(s, "{}", self.as_str()).unwrap();
    }
}
```